client/ayon_core/plugins/load/delivery.py

```diff
@@ -267,17 +266,11 @@ def deliver(self):
                     new_report_items, uploaded = deliver_single_file(*args)
                     report_items.update(new_report_items)
                     self._update_progress(uploaded)
-            else:  # fallback for Pype2 and representations without files
-                frame = repre["context"].get("frame")
-                if frame:
-                    repre["context"]["frame"] = len(str(frame)) * "#"
-
-                if not frame:
-                    new_report_items, uploaded = deliver_single_file(*args)
-                else:
-                    new_report_items, uploaded = deliver_sequence(*args)
-                report_items.update(new_report_items)
-                self._update_progress(uploaded)
+            else:
```

Comment:
I would just remove the condition `repre.get("files")`.

Reply:
Done with https://github.com/ynput/ayon-core/pull/907/commits/e9b67edc0f16f972c6d0d53f056ff69c03f49653